riff-raff/app/notification/hooks.scala

```diff
@@ -85,17 +85,18 @@ object HookConfig {
     HookConfig(UUID.randomUUID(), projectName, stage, url, enabled, new DateTime(), updatedBy)
 }
 
-object HooksClient extends LifecycleWithoutApp with Logging {
-  trait Event
-  case class Finished(uuid: UUID, params: DeployParameters)
-
+class HooksClient(wsClient: WSClient) extends Lifecycle with Logging {
   lazy val system = ActorSystem("notify")
   val actor = try {
-    Some(system.actorOf(Props[HooksClient], "hook-client"))
-  } catch { case t:Throwable => None }
+    Some(system.actorOf(Props(classOf[HooksClientActor], wsClient), "hook-client"))
```

Comment:
You could avoid the problem of this breaking silently when the constructor is refactored by using the alternative form of `Props`:

```
system.actorOf(Props(new HooksClientActor(wsClient))
```

I think there are a few other instances of `Props` elsewhere in the codebase that could be refactored in the same way.